_Week5/group1/demonstration.do

```diff
@@ -0,0 +1,19 @@
+clear
+
+// Set the working directory
+global wd "/Users/Neel Desai/Desktop/untitled folder/econ490-fall22/_Group Projects/group1/
+
+// Run the do file that contains the program
+do "$wd/one_way_anova.do"
+
+// Demonstration 1
+sysuse airfare.dta, clear
+one_way_anova airfare.dta year y98
+
+// Demonstration 2
+sysuse sandstone.dta, clear
+one_way_anova sandstone.dta depth collection
+
+// Demonstration 3
+sysuse cancer.dta, clear
+one_way_anova cancer.dta died _st
```

Comment:
Another general point that even our group discussed was whether we need to demonstrate several use cases using just one in-built dataset (that's the impression we got from the instructions) or if we can use several datasets. Given the nature of your program, I still think using several datasets (as you have done) is most appropriate since the output does not change for a given dataset unless the data are updated.

Reply:
Thanks for calling this out, Yash--it's a great point. I do think that it might make sense to demonstrate the program one multiple variables in a data set though--I'll make a note to make that change.

Reply:
@neeldesai01 just wanted to flag my reply to this comment for you